ilastik/applets/tracking/structured/structuredTrackingSerializer.py

```diff
@@ -18,102 +18,28 @@
-            if len(dset) > 0:
-                group.create_dataset(name=str(i), data=dset)
-        self.dirty = False
-
-    def deserialize(self, group):
-        if not self.name in group:
-            return
-        mainOperator = self.slot.getRealOperator()
-        innerops = mainOperator.innerOperators
-        opgroup = group[self.name]
-        for inner in opgroup.keys():
-            dset = opgroup[inner]            
-            op = innerops[int(inner)]
-            divisions = {}
-            for row in dset:
-                divisions[row[0]] = ([row[1],row[2]], row[3])
-            op.divisions = divisions
-        self.dirty = False
-        
-class SerialLabelsSlot(SerialSlot):
-    def serialize(self, group):
-        deleteIfPresent(group, self.name)
-        group = getOrCreateGroup(group, self.name)
-        mainOperator = self.slot.getRealOperator()
-        innerops = mainOperator.innerOperators
-        for i, op in enumerate(innerops):
-            gr = getOrCreateGroup(group, str(i))
-            for t in op.labels.keys():
-                t_gr = getOrCreateGroup(gr, str(t))
-                for oid in op.labels[t].keys():
-                    l = op.labels[t][oid]
-                    dset = list(l)
-                    if len(dset) > 0:
-                        t_gr.create_dataset(name=str(oid), data=dset)
-        self.dirty = False
-
-    def deserialize(self, group):
-        if not self.name in group:
-            return
-        mainOperator = self.slot.getRealOperator()
-        innerops = mainOperator.innerOperators
-        opgroup = group[self.name]
-        for inner in opgroup.keys():
-            gr = opgroup[inner]
-            op = innerops[int(inner)]
-            labels = {}
-            for t in gr.keys():
-                labels[int(t)] = {}
-                t_gr = gr[str(t)]
-                for oid in t_gr.keys():
-                    labels[int(t)][int(oid)] = set(t_gr[oid])
-            op.labels = labels
-        self.dirty = False
-        
 class StructuredTrackingSerializer(AppletSerializer):
-    
+
     def __init__(self, topLevelOperator, projectFileGroupName):
         slots = [ SerialDictSlot(topLevelOperator.Parameters, selfdepends=True),
-                 SerialHdf5BlockSlot(topLevelOperator.OutputHdf5,
-                                     topLevelOperator.InputHdf5,
-                                     topLevelOperator.CleanBlocks,
-                                     name="CachedOutput"),
                   SerialDictSlot(topLevelOperator.EventsVector, transform=str, selfdepends=True),
                   SerialDictSlot(topLevelOperator.FilteredLabels, transform=str, selfdepends=True),
+                  SerialPickledValueSlot(topLevelOperator.ExportSettings),
+                  SerialPickledValueSlot(topLevelOperator.HypothesesGraph),
+                  SerialPickledValueSlot(topLevelOperator.ResolvedMergers),
```

Comment:
By changing the StructuredTrackingSerializer this way, you break the Windows version that still uses pgmlink. Please adjust the serializer similar to https://github.com/ilastik/ilastik/blob/master/ilastik/applets/tracking/base/trackingSerializer.py, and make sure that the pgmlink version still works, too! (You can do that by removing hytra from your conda environment / renaming the hytra folder in your python site-packages)